src/types.rs

```diff
@@ -14,7 +14,8 @@ pub enum Void {}
 
 #[derive(Clone, Deserialize, Debug)]
 pub struct Config {
-    pub language: HashMap<String, LanguageConfig>,
+    #[serde(alias = "language_server")]
+    pub language: HashMap<ServerName, LanguageServerConfig>,
```

Comment:
it would probably be better to rename this to `language_server` (also in the default kak-lsp.toml) and add the alias for the legacy name.
That can all go in the parent commit.

Reply:
This has been addressed in the major commit overhaul I did.